WBooks/Screens/BookLibrary/BookLibraryViewModel.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  BookLibraryViewModel.swift
+//  WBooks
+//
+//  Created by Federico Agustin Diaz on 01/08/2019.
+//  Copyright © 2019 Wolox. All rights reserved.
+//
+
+import UIKit
+
+class BookLibraryViewModel {
+    
+    var libraryItems: [Book] = [] {
+        didSet {
+            self.reloadViewClosure?()
+        }
+    }
+    
+    var numberOfCells: Int {
+        return libraryItems.count
+    }
+    
+    var heightOfCells: CGFloat {
```

Comment:
ViewModel should have core logic, this is just something visual that you could have in a global variable or in the view controller